tests/org.jboss.tools.openshift.test/src/org/jboss/tools/openshift/test/common/core/connection/ConnectionsFactoryTest.java

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.test.common.core.connection;
+
+import java.io.IOException;
+
+import org.jboss.tools.openshift.common.core.connection.ConnectionsFactory;
+import org.jboss.tools.openshift.common.core.connection.IConnection;
+import org.jboss.tools.openshift.common.core.connection.IConnectionFactory;
+import org.jboss.tools.openshift.test.core.connection.ConnectionFake;
+
+/**
+ * @author Andre Dietisheim
+ */
```

Comment:
this is not a test class, as it has no test methods.

Tycho fails with 

      initializationError(org.jboss.tools.openshift.test.common.core.connection.ConnectionsFactoryTest)  Time elapsed: 0.002 sec  <<< ERROR!
      java.lang.Exception: No runnable methods

Either make the class abstract or you need to write actual tests